app/ide-desktop/lib/dashboard/src/pages/authentication/SetUsername.tsx

```diff
@@ -10,8 +10,11 @@ import * as backendProvider from '#/providers/BackendProvider'
 import * as textProvider from '#/providers/TextProvider'
 
 import Input from '#/components/Input'
+import AuthenticationPage from '#/components/styled/AuthenticationPage'
```

Comment:
Why do we put parts of the page that are not supposed to be used somewhere else into a common `#/components` folder, instead of putting them into a local folder near the `authentication` folder?

Reply:
good point. i guess the real issue is that the `layouts/`folder isn't really the best name for what it's used for